plugins/org.eclipse.osee.framework.ui.skynet/src/org/eclipse/osee/framework/ui/skynet/widgets/XWidget.java

```diff
@@ -231,6 +232,22 @@ public void mouseDoubleClick(MouseEvent e) {
             }
          }
       }
+      if (Widgets.isAccessible(control) && getConditions().size() > 0) {
+         for (ConditionalRule rule : getConditions()) {
+            if (this instanceof ArtifactStoredWidget && rule instanceof EnableIfCondition) {
+               if (rule.isDisabled(((ArtifactStoredWidget) this).getArtifact().getAttributesToStringList(
+                  ((EnableIfCondition) rule).getAttrType()))) {
+                  control.setEnabled(false);
+                  setEditable(false);
+                  break;
+               } else {
```

Comment:
Remove the else.  We don't want/need to enable in case some other hook or widget disabled for some other reason.